rosbridge_library/src/rosbridge_library/internal/subscribers.py

```diff
@@ -105,6 +107,7 @@ def __init__(self, topic, client_id, callback, node_handle, msg_type=None):
         self.msg_class = msg_class
         self.node_handle = node_handle
         # TODO(@jubeira): add support for other QoS.
+        print ("Subscriber in __init__: ", msg_class, "topic: ", topic, "cb: ", self.callback)
```

Comment:
Please remove the extreneous `print` logging in this module.  Use `logging` module debug level if you really want this level of verbosity.